README.md

```diff
@@ -10,11 +10,11 @@ Upon opening the app, users can pan and zoom to discover points of interest. The
 
 The app will work in many of the same ways as [whats-near.me](https://whats-near.me) (on which our project is based).
 
-For additional details, please check out the [project brief](PROJECT_BRIEF.md).
+For additional details, please check out the [project brief](PROJECT-BRIEF.md).
```

Comment:
Ugh. I'm wondering if something is still screwed up with the branches? The correct link is with the underscore, not the dash. 😬

Reply:
weird.... ok I'll just reverse these